pkg/securitycontextconstraints/sccadmission/admission_test.go

```diff
@@ -310,6 +310,7 @@ func TestAdmitSuccess(t *testing.T) {
 
 	seLinuxLevelFromNamespace := namespace.Annotations[securityv1.MCSAnnotation]
 
+	trueVal := true
```

Comment:
why is there no false or nil outcome in these tests?

Reply:
the test was set up so that it always adds a user as it's using a static SCC and a static namespace with a static UID range